R/maths_correctForMemoryEffect.R

```diff
@@ -46,6 +46,30 @@ calculateMemoryCoefficients <- function(dataset) {
   # join the mean mem coeff and the mem coeff for each standard into one dataframe
   tablesToJoin <- append(memoryCoeffForEachStandard, list(meanMemoryCoefficients))
   memCoeffOutput <- reduce(tablesToJoin, full_join, by = "Inj Nr")
+
+  # remove all trailing rows from output dataframe which only contain NA values
+  memCoeffOutput <- memCoeffOutput %>%
+    filter(cumall(!is.na(memoryCoeffD18O)))
```

Comment:
I believe that this does not work as intended. See #38

Reply:
Fixed in 2dd5f3760232a9dccf40ebbcb010effa579bc920.